sqlmesh/utils/date.py

```diff
@@ -22,24 +24,33 @@
     from sqlmesh.core.scheduler import Interval
 
 
-def now() -> datetime:
+def now(floor: bool = True) -> datetime:
```

Comment:
bad argument name. Suggestions: `round_to_minute`.

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.Timestamp.floor.html

Reply:
well in that case, there's an additional argument which specifies resolution. We don't have it there. The core of my concern here is that it's unclear to which granularity it will be floored.